src/schemas/json/haxelib.json

```diff
@@ -0,0 +1,93 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"description": "A haxelib project",
+	"type": "object",
+	"properties": {
+		"name": { "$ref": "#/definitions/projectName" },
+		"url": {
+			"description": "Project's website",
+			"type": "string",
+			"format": "uri"
+		},
+		"description": {
+			"description": "Short description of the project",
+			"type": "string"
+		},
+		"license": {
+			"description": "Open source license under which the project is licensed",
+			"enum": ["GPL", "LGPL", "BSD", "Public", "MIT", "Apache"]
```

Comment:
Yeah, right now haxelib won't allow to submit with licences that are not in this list, so this is correct.